src/index.html

```diff
@@ -7,6 +7,308 @@
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="stars__rewievs">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price-block price-block">
+            <div class="price-block__price">Price:</div>
+            <div class="price-block__amount">$2,199</div>
+          </div>
+
+          <a class="card__button" data-qa="card-hover">Buy</a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image"></div>
+
+        <div class="card__container">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="stars__rewievs">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price-block price-block">
+            <div class="price-block__price">Price:</div>
+            <div class="price-block__amount">$2,199</div>
+          </div>
+
+          <a class="card__button" data-qa="card-hover">Buy</a>
+        </div>
+      </div>
+      <div class="card">
+        <div class="card__image"></div>
+
+        <div class="card__container">
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__product-code">Product code: 195434</p>
+
+          <div class="card__stars stars">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <div class="stars__rewievs">
+              Reviews: 5
+            </div>
+          </div>
+
+          <div class="card__price-block price-block">
+            <div class="price-block__price">Price:</div>
+            <div class="price-block__amount">$2,199</div>
+          </div>
+
+          <a class="card__button" data-qa="card-hover">Buy</a>
+        </div>
+      </div>
+      <div class="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card">
```